parm/config/gefs/config.base

```diff
@@ -269,7 +269,9 @@ export MEMDIR="mem${ENSMEM}"
 
 # initialize ocean ensemble members with perturbations
 #   if true, only occurs for members greater than zero
+export END_OF_IAU_START=@END_OF_IAU_START@
```

Comment:
I think this needs to be defined for the GFS configs.

Reply:
thanks, added

Reply:
@NeilBarton-NOAA I don't see this in parm/config/gfs/config.base, maybe it didn't get pushed?